cypress/plugins/testDataProcessor.md

```diff
@@ -0,0 +1,233 @@
+# Test Data Processor Documentation
+ 
+## Overview
+This Markdown document provides documentation for the Test Data Processor. The Test Data Processor is designed to merge multiple test data JSON files located within a specified directory and resolve the value for a specified key in each object.
+ 
+ ## Functions
+ 
+### > testDataProcessor
+The Test Data Processor performs the following operations:
+- Merges all JSON files located within a specified directory.
+- Resolves the value for a specified key in each JSON object.
+
+Ex:
+
+```
+testDataProcessor()
+```
+
```

Comment:
processFireboltJson function will perform following operations 

- Iterate over each key in the provided JSON 
- Resolve the values of params, context, or content if present
- Return the JSON with the updated value.

Reply:
added the comment